src/app/app.component.ts

```diff
@@ -35,21 +38,34 @@ export class AppComponent implements AfterViewInit, OnDestroy {
 		this.mobileQuery.addListener(this.mobileQueryListener);
 	}
 
+	setSourceLinks(activeItemName: string): void {
```

Comment:
My bad , could you make instead of githubUrl method getGithubUrl() the same for sb